drivers/Makefile

```diff
@@ -30,13 +31,20 @@ DRIVERS=\
 	ip65_apple2.lib \
 	atrdragon.lib \
 	ip65_atari.lib \
+	ip65_atarixl.lib \
 	vic20rrnet.lib
 
 all: $(DRIVERS)
 
 $(DRIVERS):
 	ar65 a $@ $^
 
+atrx_timer.o: atr_timer.s
+	ca65 -t atarixl -D DYN_DRV=0 -o $@ $<
```

Comment:
Could you please elaborate in which aspect the .o files differ between `-t atari` and `-t atarixl` ?

Reply:
"-t atarixl" predefines __ATARIXL__ which in turn forces some different defines in atari.inc. ROM function addresses are mapped to low-code trampoline code which banks in the ROM before calling the requested function.

Reply:
I see. But then this seems to be true for all modules listed under `ATRX_OBJS`, isn't it? Then this building with `-t atarixl` should be applied per Makefile rule to all of them instead of explicit recipes.

Reply:
There are rules for all ATRX_OBJ files. In case you refer to ATROBJS, I don't consider the atarixl target to be an assembler target. It requires much support from the CC65 C runtime.

Reply:
Sorry, a misunderstanding. I'm wondering why we have two explicit recipes in the Makefile instead of pattern rules.

Reply:
Hmm, how would I write a pattern rule for something where in the middle there's just an extra 'x'? atrx_timer.o instead of atr_timer.o from atr_timer.s? Eager to learn... :-)